code/obj/item/organs/head.dm

```diff
@@ -251,6 +244,31 @@
 		if (src.head && src.head.wear_image_icon)
 			src.overlays += image(src.head.wear_image_icon, src.head.icon_state)
 
+		if(!(src.head && src.head.seal_hair))
+			if(src.donor_appearance?.mob_appearance_flags & HAS_HUMAN_HAIR || src.donor?.hair_override)
+				src.head_image_cust_one.pixel_x = 0
+				src.head_image_cust_one.pixel_y = 0
+				src.head_image_cust_two.pixel_x = 0
+				src.head_image_cust_two.pixel_y = 0
+				src.head_image_cust_three.pixel_x = 0
+				src.head_image_cust_three.pixel_y = 0
+				src.overlays += src.head_image_cust_one
+				src.overlays += src.head_image_cust_two
+				src.overlays += src.head_image_cust_three
+			if(src.donor_appearance?.mob_appearance_flags & HAS_SPECIAL_HAIR || src.donor?.special_hair_override)
+				src.head_image_special_one.pixel_x = 0
+				src.head_image_special_one.pixel_y = 0
+				src.head_image_special_two.pixel_x = 0
+				src.head_image_special_two.pixel_y = 0
+				src.head_image_special_three.pixel_x = 0
+				src.head_image_special_three.pixel_y = 0
+				src.overlays += src.head_image_special_one
+				src.overlays += src.head_image_special_two
+				src.overlays += src.head_image_special_three
+
+		src.pixel_y = rand(-20,-8)
+		src.pixel_x = rand(-8,8)
```

Comment:
?

Reply:
I wasn't sure what was causing the dropped head image being offset, so I moved that to the end of the head builder proc, thinking it might be doing something weird. Wasnt the case, but I think it works just the same in there, since its called when the head is dropped.